packages/billboard-client/.gitignore

```diff
@@ -1 +1,2 @@
-.env
\ No newline at end of file
+.env
+config.yml
```

Comment:
Do we have a config.yml?

Reply:
No i just copied it from be will remove this part